src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml

```diff
@@ -1,9 +1,9 @@
 @inject IAuthorizationService AuthorizationService
 <div class="dropdown">
     <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline" class="no-multisubmit">
-        <a class="ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
+        <button class="ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
```

Comment:
The rule of a button needing an inner text applies here as well, even if axe is confused by the `i` tag. Same in Layout.